Tasks/AzureFileCopyV5/MimeMapping.json

```diff
@@ -0,0 +1,564 @@
+{
+    "MIMETypeMapping": {
+     	".323": "text/h323",
+        ".3g2": "video/3gpp2",
+        ".3gp": "video/3gpp",
+        ".3gp2": "video/3gpp2",
+        ".3gpp": "video/3gpp",
+        ".7z": "application/x-7z-compressed",
+        ".aa": "audio/audible",
+        ".AAC": "audio/aac",
+        ".aaf": "application/octet-stream",
+        ".aax": "audio/vnd.audible.aax",
```

Comment:
why do we need this file?

Reply:
Removed these extensions

Reply:
@v-jkarri its not about removing some extensions. I wanted to understand where in the code this file is getting used? do we need this whole file?

Reply:
Yes Amruta. We need this  [entire file](https://github.com/Azure/azure-storage-azcopy/wiki/Custom-mime-mapping) and using this file we will override the default and OS specific mime-mappings. for that I have included all the file types.